internal/controller/suite_test.go

```diff
@@ -85,6 +86,12 @@ var _ = BeforeSuite(func() {
 
 var _ = AfterSuite(func() {
 	By("tearing down the test environment")
+	namespaceObj := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test",
+		},
+	}
+	k8sClient.Delete(context.Background(), namespaceObj)
```

Comment:
why do this after whole suite? we don't creae at start of suite do we? even if it stays what harm is it?